Traceback (most recent call last):
  File "code/functions/when-functions.py", line 10, in <module>
    turtle.mainloop()
  File "/usr/lib/python3.4/turtle.py", line 1637, in forward
    self._go(distance)
  File "/usr/lib/python3.4/turtle.py", line 1605, in _go
    self._goto(ende)
  File "/usr/lib/python3.4/turtle.py", line 3158, in _goto
    screen._pointlist(self.currentLineItem),
  File "/usr/lib/python3.4/turtle.py", line 755, in _pointlist
    cl = self.cv.coords(item)
  File "<string>", line 1, in coords
  File "/usr/lib/python3.4/tkinter/__init__.py", line 2310, in coords
    self.tk.call((self._w, 'coords') + args))]
_tkinter.TclError: invalid command name ".140241182392448"
Traceback (most recent call last):
  File "code/functions/when-functions.py", line 10, in <module>
    turtle.mainloop()
  File "/usr/lib/python3.4/turtle.py", line 1637, in forward
    self._go(distance)
  File "/usr/lib/python3.4/turtle.py", line 1605, in _go
    self._goto(ende)
  File "/usr/lib/python3.4/turtle.py", line 3158, in _goto
    screen._pointlist(self.currentLineItem),
  File "/usr/lib/python3.4/turtle.py", line 755, in _pointlist
    cl = self.cv.coords(item)
  File "<string>", line 1, in coords
  File "/usr/lib/python3.4/tkinter/__init__.py", line 2310, in coords
    self.tk.call((self._w, 'coords') + args))]
_tkinter.TclError: invalid command name ".140133472800896"
Traceback (most recent call last):
  File "code/functions/when-functions.py", line 30, in <module>
    door()
  File "code/functions/when-functions.py", line 26, in door
    t.hide()
AttributeError: 'Turtle' object has no attribute 'hide'
Traceback (most recent call last):
  File "code/functions/when-functions.py", line 30, in <module>
    door()
  File "code/functions/when-functions.py", line 26, in door
    t.hide()
AttributeError: 'Turtle' object has no attribute 'hide'
Traceback (most recent call last):
  File "code/functions/modularity.py", line 29, in <module>
    triangle(100)
NameError: name 'triangle' is not defined
Traceback (most recent call last):
  File "code/functions/modularity.py", line 29, in <module>
    triangle(100)
NameError: name 'triangle' is not defined
Traceback (most recent call last):
  File "code/functions/modularity.py", line 29, in <module>
    triangle(100)
NameError: name 'triangle' is not defined
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":0"
      after 3294 requests (3294 known processed) with 0 events remaining.
